Automattic/wp-calypso
# Domain upsell dialog: Fix track events for buttons

Related to #78429 

## Proposed Changes

The Cancel or Continue buttons on Domain upsell Dialog is not working, we fixed it

Track events:
- Open dialog: `calypso_plans_page_domain_upsell_skip_click`
- That works for me button: `calypso_plans_page_domain_upsell_cancel_plan_click`
- I want my domain as primary button: `calypso_plans_page_domain_upsell_continue_plan_click`

## Testing Instructions

* On `/home` you should see the Domain Upsell.
* Click on `Get this domain`
* On the plan's page, click `Or continue with the free plan.` on the subtitle.
* Check the track event is working.
* Click on both buttons and confirm the track events are working.

## Pre-merge Checklist
